_includes/header.html

```diff
@@ -1,9 +1,15 @@
 <header>
   <div class="pure-g">
-    <div class="pure-u-1 pure-u-lg-1-5">
-        <a href="{{ site.baseurl }}/">
-          <img class="nav-icon" src={{ site.logo }} alt="MyPaint" />
-        </a>
+    <div class="logo pure-u-1 pure-u-lg-1-5">
+      <a href="{{ site.url }}">
```

Comment:
Just letting you know, it is taking me back to the actual MyPaint website. So it probably would be a good idea to revert this back to `{{ site.baseurl }}` for development sake.